### PR TITLE
add BookHotelPage and BookHotelTest

### PR DESCRIPTION
**Pull request by AnastasiyaRahonskaya**

### CODE REVIEW

File: src/test/java/com/epam/tat/airlines/taf/page/BookHotelPage.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class BookHotelPage extends AbstractPage {
+    private final String HOME_URL = "http://www.monarch.co.uk/";
```
Comment (DzmitryZakharenka) [inline src/test/java/com/epam/tat/airlines/taf/page/BookHotelPage.java]: static

Comment (AnastasiyaRahonskaya) [inline src/test/java/com/epam/tat/airlines/taf/page/BookHotelPage.java]: done

File: src/test/java/com/epam/tat/airlines/taf/page/BookHotelPage.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class BookHotelPage extends AbstractPage {
+    private final static String HOME_URL = "http://www.monarch.co.uk/";
```
Comment (VladislavSevruk) [inline src/test/java/com/epam/tat/airlines/taf/page/BookHotelPage.java]: Modifier 'static' must appear before modifier 'final'.
Paragraph 4.8.7 of Code Style.
https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers

Comment (AnastasiyaRahonskaya) [inline src/test/java/com/epam/tat/airlines/taf/page/BookHotelPage.java]: done

**Pull request merged**